scripts/dev/build_and_deploy_operator.py

```diff
@@ -1,16 +1,16 @@
 import io
 import os
-import time
 from typing import Dict, Optional
 
 import yaml
 from kubernetes import client, config
-from kubernetes.client.rest import ApiException
 
 from dev_config import DevConfig, load_config
 from dockerfile_generator import render
 from dockerutil import build_and_push_image
 
+from k8sutil import wait_for_condition,ignore_if_already_exists, ignore_if_doesnt_exist
```

Comment:
Is this formatted properly? We need to set up a pre-commit hook for this. Generally we use [black](https://github.com/psf/black) to format our python

Reply:
Oh I didn't know this, thanks :)